R/trex.R

```diff
@@ -1,4 +1,39 @@
+#' Generate a Tableau Extension Manifest
+#'
+#' The `tableau_manifest()` function generates Tableau Extension Manifest XML
+#' which constitutes the extension manifest file (with the standard extension
+#' `.trex`). This XML text contains metadata for the extension and is used for
+#' registration.
+#'
+#' @param extension_id The ID value for the Tableau extension which follows the
+#'   reverse domain name pattern. By default, this is set to a placeholder value
+#'   of `"com.example.extensions.name"`
+#' @param extension_version The version number for the extension. The default
+#'   value for this is `"0.1.0"`.
+#' @param name The name of the extension. This name will appear as given under
+#'   *Extensions* on a *Tableau* dashboard sheet.
+#' @param description,extended_description The description and extended
+#'   description for the extension. Whereas `description` expects a relatively
+#'   short amount of text, the `extended_description` parameter serves as a
+#'   long-form description of the extension. HTML text can be used for the
+#'   latter.
+#' @param author_name,author_email,author_organization,website Details about the
+#'   extension author and project website.
+#' @param source_location The source location, which is the URL of the server
+#'   that hosts the web page that in turns interacts with Tableau.
```

Comment:
Maybe add a note that this can usually be left NULL and shinytableau will autodetect

Reply:
Added a sentence to communicate the reasoning behind the default.